sources/packages/web/src/views/aest/student/StudentDetails.vue

```diff
@@ -71,6 +71,16 @@ export default {
           });
         },
       },
+      {
+        label: "File Uploads",
+        icon: "fa fa-file-text-o",
```

Comment:
Not a part of this PR. We are using 2 different ways to define font-awesome icons one is by using array like ['fas', 'file-text-o'] and like this. IMO using the array to define will be more specific and nice. May be a topic for our dev discussion.

Reply:
@andrewsignori-aot @guru-aot @henryoforeh-aot @ann-aot ?

Reply:
Here we can't use font-awesome icons, because of the primeVue data-table,